styleguide/derek/pattern-components/headers/_headers.scss

```diff
@@ -183,7 +183,7 @@
 
 .refreshBanner-boxTop {
   display: table;
-  min-height: 70%;
+  min-height: 35%;
```

Comment:
@amberRrucker Can you make the path to all images in this file relative so images will work in Drupal?
```patch
diff --git a/styleguide/derek/pattern-components/headers/_headers.scss b/styleguide/derek/pattern-components/headers/_headers.scss
index 5f21509f..6e6a47b8 100644
--- a/styleguide/derek/pattern-components/headers/_headers.scss
+++ b/styleguide/derek/pattern-components/headers/_headers.scss
@@ -163,7 +163,7 @@
 
 .refreshBanner {
   @include make-banner;
-  background-image: url('/imgs/BG.jpg');
+  background-image: url('../imgs/BG.jpg');
   background-position: center right;
   height: 515px;
   overflow: hidden;
@@ -189,7 +189,7 @@
 
 .refreshBanner-innerTopLeft {
   @include make-angle;
-  background-image: url('/imgs/slant-topleft.svg');
+  background-image: url('../imgs/slant-topleft.svg');
   width: 16.6667%;
 }
 
@@ -219,13 +219,13 @@
 
 .refreshBanner-innerBottomLeft {
   @include make-angle;
-  background-image: url('/imgs/slant-bottomleft.svg');
+  background-image: url('../imgs/slant-bottomleft.svg');
   width: 16.6667%;
 }
 
 .refreshBanner-innerBottomRight {
   @include make-angle;
-  background-image: url('/imgs/slant-bottomright.svg');
+  background-image: url('../imgs/slant-bottomright.svg');
   width: 83.3333%;
 }
 
@@ -246,4 +246,3 @@
     width: 80%;
   }
 }
```